packages/commonwealth/docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  cw_postgres:
+    container_name: cw_postgres
+    image: postgres:15.1-alpine
+    restart: always
+    environment:
+      POSTGRES_USER: commonwealth
+      POSTGRES_PASSWORD: edgeware
+      POSTGRES_DB: commonwealth
+    volumes:
+      - .postgres:/var/lib/postgresql/data
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U commonwealth']
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    ports:
+      - '5432:5432'
+    networks:
+      - cw-network
+
+  cw_rabbitmq:
+    container_name: cw_rabbitmq
+    image: rabbitmq:3.11-management
+    restart: always
+    # environment:
+    #   RABBITMQ_DEFAULT_USER: commonwealth
+    #   RABBITMQ_DEFAULT_PASS: edgeware
+    volumes:
+      - '.rabbitmq:/var/lib/rabbitmq/mnesia/'
+    ports:
+      - 5672:5672
+      - 15672:15672
+    networks:
+      - cw-network
+
```

Comment:
Redis should be added to the RMQ file. Although the app will run without it, we plan to merge Nakul's PRs in soon which will cache certain things in it for better performance.

To do this add to services:
```
loacl-redis:
    container_name: local-redis
    image: redis:latest
    ports:
      - 6379:6379
    volumes:
      - ./redis-data:/data
  ```

Reply:
Amazing, thanks Kurtis

Reply:
we can flag this as an optional strategy, I know windows users will have to tweak this a bit 😅

Reply:
@Rotorsoft any reason why windows users would need to tweak it? In my understanding we just start up a redis container, and persist its volume to the ./redis-data directory.

Reply:
I was talking about docker in general, not the specific redis config... but this shouldn't stop this PR. We should all have some basic understanding of how to run docker-compose in our local env and be able to adjust accordingly.